parsec/core/gui/file_history_widget.py

```diff
@@ -161,4 +162,5 @@ def exec_modal(
         d = GreyedDialog(
             w, title=_("TEXT_FILE_HISTORY_TITLE_name").format(name=path.name), parent=parent
         )
+        w.dialog = d
```

Comment:
Wouldn't it better to create GreyedDialog in `FileHistoryWidget.__init__` so that `FileHistoryWidget.dialog` can never be `None` ?

Reply:
It might. I originally designed these widgets to be independant (showing them without a GreyedDialog). Later on when GreyedDialog was added, it seemed easier to add a classmethod to do the job rather than update the widget itself.
It would also be weird to have a widget construct itself and its parent in the __init__. I'm not very satisfied with this dialog variable but it is, imo, the best choice.